frontend/packages/noobaa-storage-plugin/src/components/status-card/status-card.tsx

```diff
@@ -68,62 +68,46 @@ const StatusCard: React.FC<DashboardItemProps> = ({
   React.useEffect(() => {
     watchK8sResource(noobaaResource);
     statusCardQueries.forEach((key) => watchPrometheus(StatusCardQueries[key]));
-    watchPrometheus(DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY);
     return () => {
       stopWatchK8sResource(noobaaResource);
       statusCardQueries.forEach((key) => stopWatchPrometheusQuery(StatusCardQueries[key]));
-      stopWatchPrometheusQuery(DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY);
     };
   }, [watchK8sResource, stopWatchK8sResource, watchPrometheus, stopWatchPrometheusQuery]);
 
-  const buckets = prometheusResults.getIn([
-    StatusCardQueries.BUCKETS_COUNT,
+  const healthStatusResult = prometheusResults.getIn([
+    StatusCardQueries.HEALTH_QUERY,
     'data',
   ]) as PrometheusResponse;
 
-  const unhealthyBuckets = prometheusResults.getIn([
-    StatusCardQueries.UNHEALTHY_BUCKETS,
-    'data',
-  ]) as PrometheusResponse;
-
-  const pools = prometheusResults.getIn([
-    StatusCardQueries.POOLS_COUNT,
+  const progressResult = prometheusResults.getIn([
+    StatusCardQueries.REBUILD_PROGRESS_QUERY,
     'data',
   ]) as PrometheusResponse;
 
-  const unhealthyPools = prometheusResults.getIn([
-    StatusCardQueries.UNHEALTHY_POOLS,
-    'data',
+  const healthStatusError = prometheusResults.getIn([
+    StatusCardQueries.HEALTH_QUERY,
+    'loadError',
   ]) as PrometheusResponse;
 
-  const progressResult = prometheusResults.getIn([
-    DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY,
-    'data',
-  ]) as PrometheusResponse;
   const progressError = prometheusResults.getIn([
-    DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY,
+    StatusCardQueries.REBUILD_PROGRESS_QUERY,
     'loadError',
   ]);
 
   const noobaa = _.get(resources, 'noobaa') as FirehoseResult;
 
-  const queriesLoadError = statusCardQueries.some((q) => {
-    return prometheusResults.getIn([StatusCardQueries[q], 'loadError']);
-  });
-
-  const hasStatusLoadError = _.get(noobaa, 'loadError') || queriesLoadError;
-  const allStatusLoaded =
-    _.get(noobaa, 'loaded') && buckets && unhealthyBuckets && pools && unhealthyPools;
+  const hasStatusLoadError = _.get(noobaa, 'loadError') || healthStatusError;
+  const allStatusLoaded = _.get(noobaa, 'loaded') && healthStatusResult;
```

Comment:
please add types since _.get will return any

Reply:
loaderror returns a typerror object, typically 'any`.
I can add for loaded that is known to be boolean.

Ref: [#comment](https://github.com/openshift/console/pull/2897#discussion_r332027360)